platform/admin/core/admin-core-logviewer/README.md

```diff
@@ -0,0 +1,58 @@
+<!--
```

Comment:
Are we using something to generate the standard javascript project structure? i.e. yoeman